### DISCUSSION TITLE
Animated QR code (Safe start)

### DISCUSSION THREAD

**Discussion by Max-Eriksson** [ActiveLogin/ActiveLogin.Authentication#361] [Q&A] [answered]
Hi,

According to the [documentation](https://github.com/ActiveLogin/ActiveLogin.Authentication#features) animated QR code (Safe start) is supported when using BankId. How do you activate animated qr codes? We're using the `UseQrCoderQrCodeGenerator` extension method to enable qr codes but that renders a static (not animated) qr code. We're using version 4.0.0. Is animated qr code a 5.0.0 feature?

BR,
Max.

**Comment by PeterOrneholm** [comment, marked as answer]
Hi Max!

Yes, that is correct. The animated QR codes feature was released as part of our version 5.0.0. See the release notes for [v5.0.0](https://github.com/ActiveLogin/ActiveLogin.Authentication/releases/tag/v5.0.0).

As static QR codes are deprecated, we only support animated QR-codes from now on. So by enabling QR codes using `UseQrCoderQrCodeGenerator()` you will get them automatically.

Does this answer your question?

/Peter
